test-network-function/lifecycle/suite.go

```diff
@@ -107,7 +131,7 @@ var _ = ginkgo.Describe(common.LifecycleTestKey, func() {
 
 		testGracePeriod(env)
 
-		testShutdown(env)
+		test(env)
```

Comment:
"test" seems too generic word, and feels a bit inconsistent with the surrounding calls to functions whose name refer to single test cases. Leave the old testShutdown() function and add a new testReadiness() and testLiveness() functions.

Reply:
I agree that word "test" its not good but when I create 2 function - 1 for liveness and 1 for readiness , I getting error in "make lint" that have lines duplicate.
you can to see this in previous commit.